### PR TITLE
Add a docs build context and add it to the builds if there are docs changes in the PR

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

@jfrazelle I'm not totally sure if this is right - my leeroy/jenkins instance has issues with the DCO checking, and also doesn't have a representative set of jenkins tasks.

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@jfrazelle @calavera @moxiegirl

**Comment by moxiegirl** [comment]
@SvenDowideit is there a way to test this? Can I like do a PR With a .go file in tutorials and it will ignore it?

File: Dockerfile.build [hunk 1]
```diff
@@ -2,10 +2,18 @@ FROM golang:latest
 MAINTAINER Jess Frazelle <jess@docker.com>
 
 RUN go get github.com/Sirupsen/logrus && \
-    go get github.com/crosbymichael/octokat
+    go get github.com/crosbymichael/octokat && \
+	go get github.com/tobi/airbrake-go && \
+	go get github.com/bugsnag/bugsnag-go && \
+	go get github.com/getsentry/raven-go && \
+	go get gopkg.in/gemnasium/logrus-airbrake-hook.v2 && \
+	go get github.com/revel/revel && \
```
Comment (jfrazelle) [inline Dockerfile.build]: why all these

Comment (SvenDowideit) [inline Dockerfile.build]: means that I'm not constantly re-fetching the req's after the COPY. (my internet is unfun, so I use the build cache lots.

**Pull request merged**